src/Response/Header/XFrameOptions.php

```diff
@@ -0,0 +1,77 @@
+<?php
+namespace FMUP\Response\Header;
+
+use FMUP\Response\Header;
+
+class XFrameOptions extends Header
+{
+    const TYPE = 'X-Frame-Options';
+
+    const OPTIONS_DENY = 'Deny';
+    const OPTIONS_SAMEORIGIN = 'Sameorigin';
+    const OPTIONS_ALLOW_FROM = 'ALLOW_FROM';
+    const OPTIONS_ALLOW_FROM_URI_DEFAULT = '*';
+
+    private $options = self::OPTIONS_DENY;
+    private $uri = self::OPTIONS_DENY;
+
+    /**
+     * @param bool $options
+     */
+    public function __construct($options,$uri = array())
+    {
+        $this->setOptions($options);
+        $this->setURI($uri);
+    }
+
+    public function setOptions($options){
+       $this->options = $options;
+       return $this;
+    }
+
+    public function getOptions(){
+        return $this->options;
+    }
+
+
+    public function setURI($uri)
+    {
```

Comment:
Please make sure `setUri` is camelCase and `$uri` match the right type instead of mixed